layouts/blog/rss.xml

```diff
@@ -0,0 +1,29 @@
+<rss version="2.0">
```

Comment:
Consider adding the following as the first line:

```
{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\" ?>" | safeHTML }}
```

Per https://gohugo.io/templates/rss/